README.rst

```diff
@@ -1,4 +1,4 @@
-.. image:: https://circleci.com/gh/ARMmbed/mbed-cli.svg?style=svg
+## Arm Mbed CLI
```

Comment:
This isn't correct restructured text formatting, it's markdown. However, PyPI now supports markdown so we can combine the readmes in a single markdown file which works in both GitHub and PyPI.